doc/source/release-notes/Percona-Server-5.6.39-83.1.rst

```diff
@@ -0,0 +1,70 @@
+.. rn:: 5.6.39-83.1
+
+=============================
+|Percona Server| 5.6.39-83.1
+=============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.39-83.1 on
+February 13, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.39-83.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.6.39
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-39.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.39-83.1 is now the current
+stable release in the 5.6 series. All of |Percona|'s software is open-source
+and free.
+
+Bugs Fixed
+==========
+
+With :variable:`innodb_large_prefix` set to ``1``, Blackhole storage engine
+was incompatible with InnoDB keys longer than 1000 bytes, thus adding new
+indexes would cause replication errors on the slave. Bug fixed :psbug:`1126`
+(upstream :mysqlbug:`53588`).
+
+Intermediary slave with Blackhole storage engine couldn't record updates
+from master to its own binary log in case master has
+:variable:`binlog_rows_query_log_events` option enabled. Bug fixed
+:psbug:`1119` (upstream :mysqlbug:`88057`).
+
+A build error on FreeBSD caused by :psbug:`2284` was fixed.
```

Comment:
It looks like you have replaced the fixed bug id with the bug id whose change introduced the regression, but both have to be present